eks-distro-base/tests/run_tests.sh

```diff
@@ -39,10 +61,16 @@ function check_base() {
         --opt target=check-base \
         --local dockerfile=./ \
 		--local context=./tests \
-		--output type=image,oci-mediatypes=true,\"name=$LOCAL_REGISTRY/base-test:latest\",push=true
+        --export-cache type=inline \
+        --import-cache type=registry,ref=$LOCAL_REGISTRY/eks-distro-minimal-images-base-test:base-latest \
+        --output type=image,oci-mediatypes=true,\"name=$LOCAL_REGISTRY/eks-distro-minimal-images-base-test:base-latest\",push=true
 
     for platform in ${PLATFORMS//,/ }; do
-        docker run --rm --platform=$platform $LOCAL_REGISTRY/base-test:latest
+        build::docker::retry_pull --platform=$platform $LOCAL_REGISTRY/eks-distro-minimal-images-base-test:base-latest
```

Comment:
When building to private ecr and not being auth'd to public.ecr, you can get rate limited. These changes are all just added a pull with retry before calling run